### ISSUE TITLE
Resurrect (or at least try!) the Heavy Gear

### ISSUE THREAD

**Issue by Lisias** [net-lisias-ksp/KAX#11]
As the tittle says.

Probably I'm going to need KSP-Wheels, as I doubt the current mesh will cope with the modern Wheel's modules.

**Comment by Lisias** [comment]
Since the Gear is longtime missing from the main distribution, it's not a bug but an enhancement.

**Comment by Lisias** [comment]
TL;DR: I would love to have the FS Wheels working again. :)

The Gears are working **fine** on KSP 1.3, and I don't have the slightest clue about the reason they where withdrawn for the KSP 1.1.2 (2016, Version 2.6.2). Since KSP 1.4 screwed up FS Wheels royally, I would understand it being withdrawn on 1.4.0, but I fail to reach the reason for removing it on 1.1.2.

Using the Stock Wheels appears to be some work, as the meshes should be tied to the modules and I would have to rework them. Appears to be plausible, I will try to figure my way on newer Blender.

KSP Wheels appears to be highly configurable and versatile - but I confess I'm reticent on adding yet another dependency to KAX. And I need to learn my way on it.

The best possible solution would be to fix the FS Wheels but, even if I manage to do that, getting the P/R accepted on the upstream will be an impossible mission, and so I would need to fork FirespitterCore at once (and then fight my way into CKAN, SpaceDock and Forum) - or just create yet another "Add'On" with the fixes implemented on new named modules - and so I would have a new mouth to feed. And a new KAX dependency anyway.

Choices. Sometimes, we have them too many.
